metrics-recorder-prometheus/src/lib.rs

```diff
@@ -1,12 +1,15 @@
 //! Records metrics in the Prometheus exposition format.
+#![deny(missing_docs)]
 use hdrhistogram::Histogram;
 use metrics_core::{Key, Recorder};
 use metrics_util::{parse_quantiles, Quantile};
+use std::collections::HashMap;
```

Comment:
I see that you already have `hashbrown` as a dependency for you runtime. Why not use it everywhere since its basically a faster implementation?

Reply:
Yeah, that seems fine to do.  I'll get that done.

Reply:
All set.